suricata.yaml.in

```diff
@@ -819,6 +819,10 @@ app-layer:
       #hassh: yes
     http2:
       enabled: yes
+      # Maximum number of live HTTP2 streams in a flow
+      #max-streams: 4096
```

Comment:
Where should I document these ?

Reply:
cc @jufajardini

Reply:
Should I create a ticket for adding that to the userguide?

Reply:
https://redmine.openinfosecfoundation.org/issues/4949 Not sure if should be assigned or not...

Reply:
I can do this as part of this PR.
But I was asking where in the user guide I should document these.
Juliana, would you know where ?

Reply:
As things are in the documentation now, I would go with Configuration >  suricata.yaml > application layer parsers (https://suricata.readthedocs.io/en/latest/configuration/suricata-yaml.html#application-layer-parsers).

Would that make sense, @catenacyber ?